inc/ThirdParty/Plugins/PageBuilder/BeaverBuilder.php

```diff
@@ -32,4 +32,16 @@ public function purge_cache() {
 		rocket_clean_minify();
 		rocket_clean_domain();
 	}
+
+	/**
+	 * Purge cache of a page when it is updated.
+	 *
+	 * @param int $post_id post ID.
+	 *
+	 * @return void
+	 */
+	public function purge_page( $post_id ) {
+		rocket_clean_minify();
```

Comment:
If you clean the minified files without clearing the full cache, and CSS/JS combine are active, some pages might end up with 404 on minified files because they will reference deleted files